<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransPrintMarginsDialog</class>
 <widget class="QDialog" name="TransPrintMarginsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,4,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Marges (mm)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>19</number>
        </property>
        <property name="rightMargin">
         <number>19</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Haut</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinT">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>37</width>
            <height>0</height>
           </size>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinL">
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Gauche</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Droite</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinR">
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="leftMargin">
         <number>19</number>
        </property>
        <property name="rightMargin">
         <number>19</number>
        </property>
        <item>
         <widget class="QSpinBox" name="spinB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>37</width>
            <height>0</height>
           </size>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Bas</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Police</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFontComboBox" name="Fonts">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="writingSystem">
         <enum>QFontDatabase::Latin</enum>
        </property>
        <property name="currentFont">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="fontSizeBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="prefix">
           <string>taille </string>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ClickableQLabel" name="konami_label">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boutOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutAN">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableQLabel</class>
   <extends>QLabel</extends>
   <header>clickableqlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinL</tabstop>
  <tabstop>spinT</tabstop>
  <tabstop>spinR</tabstop>
  <tabstop>spinB</tabstop>
  <tabstop>Fonts</tabstop>
  <tabstop>boutOK</tabstop>
  <tabstop>boutAN</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
